go/oasis-node/cmd/signer/signer.go

```diff
@@ -1,61 +0,0 @@
-// Package signer registers all subcommands needed by specific signers
-package signer
-
-import (
-	"os"
-
-	"github.com/spf13/cobra"
-
-	"github.com/oasisprotocol/oasis-core/go/common/logging"
-	cmdCommon "github.com/oasisprotocol/oasis-core/go/oasis-node/cmd/common"
-	cmdSigner "github.com/oasisprotocol/oasis-core/go/oasis-node/cmd/common/signer"
-	"github.com/oasisprotocol/oasis-core/go/oasis-node/cmd/signer/ledger"
-)
-
-var (
-	signerCmd = &cobra.Command{
-		Use:   "signer",
-		Short: "signer backend utilities",
-	}
-
-	exportCmd = &cobra.Command{
-		Use:   "export",
-		Short: "export the public key from signer as an empty entity",
-		Run:   doExport,
-	}
-
-	logger = logging.GetLogger("cmd/signer")
-)
-
-func doExport(cmd *cobra.Command, args []string) {
```

Comment:
Is this no longer needed or is this something that should be moved to the signer-specific cmd? Seems generic though.

Reply:
Oh whoops.  Forgot we had that.